ProcessInProcess: Updating Github URLs

ahori opened the pull request:
Process-in-process repo moves to new Github.  This PR updates the URLs.  Additionally PiP test is moved to the very end of installation.

Commit:
Update ichange git url and move PiP-Testsuite to the very end)

Commit:
Merge branch 'develop' of github.com:spack/spack into develop

Commit:
Merge branch 'develop' of github.com:spack/spack into develop

Commit:
removing a trailing space

ahori commented:
Ping.  Am I supposed to do something?

haampie commented on var/spack/repos/builtin/packages/process-in-process/package.py:
```diff
@@ -35,47 +37,47 @@ class ProcessInProcess(Package):
     #   for rhel/centos 7
     resource(name='PiP-glibc',
-             git='https://github.com/RIKEN-SysSoft/PiP-glibc.git',
+             git='git@github.com:procinproc/PiP-glibc.git',
```
Can you leave it at https? Otherwise it requires ssh authentication.

ahori replied:
Oh, yes, you are right.  I will fix this.  

BTW, is it better to use hub?

Commit:
haampie:
Can you leave it at https? Otherwise it requires ssh authentication.

This is the fix.

Commit:
Fix: missing whitespace around arithmetic operator
